src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -513,8 +513,8 @@
     <value>Exception occurred with following context:
 {0}</value>
   </data>
-  <data name="TupleNamesAllOrNone" xml:space="preserve">
-    <value>If names are used in a tuple type, then there must be the same number of names and elements.</value>
+  <data name="A_tuple_of_0_elements_can_have_0_names_or_none_but_not_1" xml:space="preserve">
```

Comment:
ElementNameCountMismatch   ?

"If names are specified, the length of the 'names' array must match the element count of the tuple. "